### PR TITLE
NO-JIRA deprecation fixes

### PR DESCRIPTION
**Pull request by jdanekrh**
This is kind of a followup to #1220. I am putting each fix into separate commit for easier review (which is probably not necessary anymore, because the previous PR fixed a lot of what I originally had prepared here too, so there is not much to review).

I am wondering. What is the opinion about using `//noinspection deprecation` magic comment that IntelliJ likes? (It can apply to a single statement, while the annotation form `@SuppressWarnings("deprecation")` cannot.)

### CODE REVIEW

**Comment by jdanekrh** [comment]
Same as previous PR, Jenkins failed because of 

>Tests in error: 
>  WebServerComponentTest.simpleServer:87 » Bind Address already in use
>  WebServerComponentTest.testComponentStopBehavior:124 » Bind Address already in...

File: artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/impl/ClientSessionFactoryImpl.java [hunk 1]
```diff
@@ -338,7 +338,7 @@ public ClientSession createSession(final boolean xa,
       return createSessionInternal(null, null, xa, autoCommitSends, autoCommitAcks, preAcknowledge, serverLocator.getAckBatchSize());
    }
 
-   // ConnectionLifeCycleListener implementation --------------------------------------------------
```
Comment (clebertsuconic) [inline artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/impl/ClientSessionFactoryImpl.java]: where is the deprecation? I see this a fix, but I'm not sure why you're calling this a deprecation?

File: artemis-server/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/NettyAcceptor.java [hunk 2]
```diff
@@ -300,7 +300,7 @@ public synchronized void start() throws Exception {
       if (useInvm) {
          acceptorType = INVM_ACCEPTOR_TYPE;
          channelClazz = LocalServerChannel.class;
-         eventLoopGroup = new LocalEventLoopGroup();
+         eventLoopGroup = new DefaultEventLoopGroup();
```
Comment (clebertsuconic) [inline artemis-server/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/NettyAcceptor.java]: nice one

File: artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/impl/ClientSessionFactoryImpl.java (hunk 1 above)
Comment (clebertsuconic) [inline artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/impl/ClientSessionFactoryImpl.java]: oic

File: artemis-server/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/NettyAcceptor.java (hunk 2 above)
Comment (jdanekrh) [inline artemis-server/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/NettyAcceptor.java]: thanks. :)

**Pull request closed without merging**